docs/site/Model.md

```diff
@@ -190,9 +188,198 @@ class Product extends Entity {
 }
 ```
 
-Additionally, the model decorator is able to build the properties object through
-the information passed in or inferred by the property decorators, so the
-properties key value pair can also be omitted.
+However, the model decorator in lb4 is not exactly the same as what it is in
+lb3. For example, in
+[lb3](https://loopback.io/doc/en/lb3/Model-definition-JSON-file.html) we can
+pass in a model definition syntax in model decorator, such as properties,
+options, relation etc. But not all these entries are available in lb4 model
+decorator:
+
+In lb4, we only support `properties` and `setting` in the ModelDefinitionSyntax
+for now. Those `top-level properties` in lb3 now are passed in `settings`.
+[Options](https://loopback.io/doc/en/lb3/Model-definition-JSON-file.html#options)
+in lb3 doesn't have the mapping feature in lb4 yet.
+
+As for entries in `settings`, lb4 supports these built-in entries for now:
+
+#### Available Entries of Settings
+
+  <table>
+  <thead>
+  <tr>
+    <th width="160">Property</th>
+    <th width="100">Type</th>
+    <th>Default</th>
+    <th>Description</th>
+  </tr>
+  </thead>
+
+  <tbody>
+  <tr>
+  <td><code>base</code></td>
+  <td>String</td>
+  <td>None</td>
+  <td>
+    Name of another model that this model extends. The model will "inherit" properties and methods of the base model.
+  </td>
+  </tr>
+
+  <tr>
+  <td><code>description</code></td>
+  <td>String or Array</td>
+  <td>None</td>
+  <td>
+    Optional description of the model.<br/><br/>
+    You can split long descriptions into arrays of strings (lines) to keep line lengths manageable; for example:
+  <pre style="font-size: 75%;">[ "Lorem ipsum dolor sit amet, consectetur adipiscing elit",
```

Comment:
I am not sure if we support description yet, for example whether it's copied to OpenAPI/JSON schema emitted for models. Having said that, I think it's a good idea to document that field :+1:

I have mixed feeling about the "arrays of string" flavor. I have introduced it to make it easier to create long descriptions in JSON files, however it was not welcome by groups working on Swagger and OpenAPI specs.

Let's remove this form from our docs for now. We can revisit this decision later, if there is any user demand.

Reply:
> for example whether it's copied to OpenAPI/JSON schema emitted for models.

Not really. Good to know the model description could map to the schema's description....

Reply:
> Let's remove this form from our docs for now. We can revisit this decision later, if there is any user demand.

Just to clarify - I am proposing to remove support for `description` set to an array of strings. The `description` field itself should stay supported & documented.

> > for example whether it's copied to OpenAPI/JSON schema emitted for models.
>
> Not really.

Let's open a new GH issue to describe that feature, then we can mention the issue here in the docs.